bitnami/rabbitmq/templates/servicemonitor.yaml

```diff
@@ -9,20 +9,12 @@ kind: ServiceMonitor
 metadata:
   name: {{ include "common.names.fullname" . }}
   namespace: {{ default (include "common.names.namespace" .) .Values.metrics.serviceMonitor.namespace | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-    {{- if .Values.metrics.serviceMonitor.labels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.serviceMonitor.labels "context" $ ) | nindent 4 }}
-    {{- end }}
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-    {{- end }}
-  annotations:
-    {{- if .Values.commonAnnotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-    {{- end }}
-    {{- if .Values.metrics.serviceMonitor.annotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.serviceMonitor.annotations "context" $ ) | nindent 4 }}
-    {{- end }}
+  {{- $labels := merge .Values.metrics.serviceMonitor.labels .Values.commonLabels }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" $labels "context" $ ) | nindent 4 }}
+  {{- if or .Values.serviceMonitor.annotations .Values.commonAnnotations }}
+  {{- $annotations := merge .Values.serviceMonitor.annotations .Values.commonAnnotations }}
```

Comment:
Was this supposed to be `.Values.metrics.serviceMonitor.annotations`? Installation fails with this change for me:

```
Error: template: rabbitmq/templates/servicemonitor.yaml:14:19: executing "rabbitmq/templates/servicemonitor.yaml" at <.Values.serviceMonitor.annotations>: nil pointer evaluating interface {}.annotations
```